packages/box/test.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable no-undef */
+import tap, { test, beforeEach, teardown } from 'tap';
+import { chromium } from 'playwright';
+
+tap.before(async () => {
+  const browser = await chromium.launch({ headless: true });
+  tap.context.browser = browser;
+});
+
+beforeEach(async (t) => {
+  const { browser } = t.context;
+  const context = await browser.newContext();
+  t.context.page = await context.newPage();
+});
+
+teardown(async () => {
+  const { browser } = tap.context;
+  browser.close();
+});
+
+test('Box component with no attributes is rendered on the page', async (t) => {
+  // GIVEN: A box component
+  const component = `
+    <f-box>
+      <p>This is a box</p>
+    </f-box>
+  `;
+
+  // WHEN: the component is added to the page
+  const { page } = t.context;
+  await page.setContent(component);
+  await page.addScriptTag({ path: './dist/index.js', type: 'module' });
```

Comment:
As per our early discussion, these 3 lines would probably be a good candidate for extraction into a common function. No need to do this now unless you want to but I guess we could add a common test-utils file with a `fixture` function that returns a page? Something like:

```js
await fixture(t.context.page, component);
```

Reply:
Refactored both `card` and `box` tests using a function I called `addContentToPage` in [5672d26](https://github.com/fabric-ds/elements/pull/68/commits/5672d261b1586519bf2f6bc2047483bac57e43eb), but maybe the name is too specific? I didn't quite understand what `fixture` means 😅